### ISSUE TITLE
Improve test coverage

### ISSUE THREAD

**Issue by DAddYE** [padrino/padrino-integration#4]
1) Create a new model
2) Create the admin page
3) Create the frontend page

@nesquena, @achiu any other idea?

**Comment by nesquena** [comment]
Can we test development reloading, i think so.
Creating a controller and model and visiting the controller and checking data is installed.
Creating two subapps and testing that session data can be passed between them in a freshly generated project

**Comment by DAddYE** [comment]
Let's go @nesquena start to improve it ;)

**Comment by nesquena** [comment]
Yep I plan to start adding more tests to this, as we all should as time permits.

**Comment by nesquena** [comment]
More ideas, let's have a sinatra app with module included (helpers) and then make sure for every template engine that rendering a template with a partial works.

**Comment by DAddYE** [comment]
@nesquena extend this: https://github.com/padrino/single-apps then we can add test for every thing ;)

**Comment by DAddYE** [comment]
Now tests are more more fast.

Now tests are in rspec for a better output and log backtrace

Added spec for single-apps, @nesquena @achiu ? Can we daily improve our coverage? Requires no more than 10 minutes write few lines.

**Comment by DAddYE** [comment]
I forgot a big mention! Now every thing use our internal padrino reloader, I think there is no way better than this to check that every thing works like a charm!

**Comment by nesquena** [comment]
Awesome, great work man!

**Comment by dariocravero** [comment]
See padrino/padrino-framework#920.

**Comment by wikimatze** [comment]
Can we delete this repository? Integration testing is done on travis.
